jquery.flot.js

```diff
@@ -1325,7 +1325,7 @@ Licensed under the MIT license.
             var magn = Math.pow(10, -dec),
                 norm = delta / magn;
 
-            if (norm > 2.25 && norm < 3 && (tickDecimals == null || dec + 1 <= tickDecimals)) {
+            if (tickDecimals != null && norm > 2.25 && norm < 3 && (dec + 1) <= tickDecimals) {
```

Comment:
I'm wondering whether you could refactor this function to perform a single null check for the tickDecimals argument at the beginning of the function. Validating the refactoring work should be piece of cake since it already has a high code coverage, does it? :-P